Cargo.toml

```diff
@@ -21,10 +21,10 @@ harfrust = "0.1.2"
 png = "0.17.16"
 tiny-skia = "0.11.4"
 roxmltree = "0.20"
+read-fonts = "0.33.1"
```

Comment:
Is this necessary? Doesn't skrifa include/forward read-fonts? Is there a version mismatch?